sequencer/src/state.rs

```diff
@@ -229,9 +227,35 @@ impl hotshot_types::traits::states::TestableState for ValidatedState {
 pub type BlockMerkleTree = LightWeightSHA3MerkleTree<Commitment<Header>>;
 pub type BlockMerkleCommitment = <BlockMerkleTree as MerkleTreeScheme>::Commitment;
 
+#[derive(
+    Default,
+    Hash,
+    Copy,
+    Clone,
+    Debug,
+    Deserialize,
+    Serialize,
+    PartialEq,
+    Eq,
+    CanonicalSerialize,
+    CanonicalDeserialize,
+)]
+/// `FeeInfo` holds data related to builder fees.
+pub struct FeeInfo {
+    account: FeeAccount,
+    amount: FeeAmount,
+}
+impl FeeInfo {
+    pub fn new(account: FeeAccount) -> Self {
+        let amount = FeeAmount::default(); // TODO grab from config (instance_state?)
```

Comment:
Leaving this 0 is fine for now. Eventually we will have third party builders and each one will decide how to set it.